src/test/java/io/jenkins/plugins/checks/ArchitectureTest.java

```diff
@@ -1,33 +0,0 @@
-package io.jenkins.plugins.checks;
-
-import com.tngtech.archunit.junit.AnalyzeClasses;
-import com.tngtech.archunit.junit.ArchTest;
-import com.tngtech.archunit.lang.ArchRule;
-
-import edu.hm.hafner.util.ArchitectureRules;
-
-import io.jenkins.plugins.util.PluginArchitectureRules;
-
-/**
- * Defines several architecture rules for the static analysis model and parsers.
- *
- * @author Ullrich Hafner
- */
-@SuppressWarnings("hideutilityclassconstructor")
-@AnalyzeClasses(packages = "io.jenkins.plugins..")
-class ArchitectureTest {
-    @ArchTest
-    static final ArchRule NO_JENKINS_INSTANCE_CALL = PluginArchitectureRules.NO_JENKINS_INSTANCE_CALL;
```

Comment:
Whatever this stuff was doing, it can presumably be dropped without anyone noticing.